common/changes/@itwin/viewer-react/leck-tool-settings_2022-07-19-15-40.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/viewer-react",
+      "comment": "Hide tool settings by default",
```

Comment:
sorry i was rushing yesterday and typed the wrong thing for the issue. its not that we want to hide tool settings by default, but we want to expose an option to do so